install/kubernetes/helm/istio/charts/telemetry-gateway/templates/gateway.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or (.Values.prometheusEnabled) (.Values.grafanaEnabled) }}
+{{- if or (.Values.prometheusEnabled) (.Values.grafanaEnabled) (.Values.tracingEnabled)}}
```

Comment:
Looks pretty good.  I would however recommend just moving the gateway logic for jaeger to the jaeger-specific chart.  Easier to read and maintain.  I'd also like to move this specific logic for grafana or prometheus to their respective charts (although this isn't possible with grafana which is why this hasn't been done..

In the case of